src/sentry/snuba/discover.py

```diff
@@ -726,6 +735,7 @@ def top_events_timeseries(
                 auto_aggregations=True,
                 use_aggregate_conditions=True,
                 use_snql=use_snql,
+                auto_equation_fields=True,
```

Comment:
should we add a test case in test_discover.py to cover this change?

Reply:
I added a test to assert that this is called with `include_equation_fields=True`. I'm not sure if that's what you had in mind but we can follow up tomorrow if not and I can write a proper test.